components/blitz/resources/omero/api/RawPixelsStore.ice

```diff
@@ -41,14 +41,17 @@ module omero {
 
                 /**
                  * Initializes the stateful service for a given Pixels set.
+                 *
                  * @param pixelsId Pixels set identifier.
-                 * @param bypassOriginalFile Whether or not to bypass checking for an
-                 * original file to back the pixel buffer used by this service. If requests
-                 * are predominantly <code>write-only</code> or involve the population of
-                 * a brand new pixel buffer using <code>true</code> here is a safe
-                 * optimization otherwise <code>false</code> is expected.
+                 * @param bypassOriginalFile Whether or not to bypass checking
+                 *        for an original file to back the pixel buffer used
+                 *        by this service. If requests are predominantly
+                 *        <code>write-only</code> or involve the population of
+                 *        a brand new pixel buffer using <code>true</code>
+                 *        here is a safe optimization otherwise
+                 *        <code>false</code> is expected.
                  *
-                 * See "read-only caveat" under {@link RawPixelsStore}
+                 * See "Read-only caveat" under {@link RawPixelsStore}
```

Comment:
For some reason I am not seeing this change in the rendered docs. But regardless of that does this change need to be made elsewhere in this file? Line 270 onwards has a few occurrences.